src/components/FormRenderer/components/Custom/index.tsx

```diff
@@ -15,12 +15,13 @@
  ******************************************************************************************************************** */
 import React, { FunctionComponent, memo } from 'react';
 import { useFieldApi, useFormApi } from '@data-driven-forms/react-form-renderer';
+import ReactMarkdown from 'react-markdown';
 
-const CustomMapping: FunctionComponent = (props: any) => {
+const MarkdownViewer: FunctionComponent = (props: any) => {
```

Comment:
Should we create a new mapping component here called Markdown instead of change the Csutom component?

Reply:
this was removed in this PR and will be created when i build the MarkdownEditor component